### ISSUE TITLE
Continue gathering information on coded prototypes

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#15874]
## Description
Follow up to #15543

Driving factor: how can we make a coordinated effort across OCTO so there is little wasted effort and possibly contribute back to how accessible prototyping can be successful?

In order to gather more information for this, @aklausmeier  and Laura  would like to meet with other VA teams to discuss:

- how they decided on staging vs. Codepen prototyping
- the process/implementation
- any recent usability studies using the prototype 

## Acceptance Criteria
- [x] Meet with the following teams:
    - [x] Veteran Facing forms
    - [x] Check-in

**Comment by laflannery** [comment]
### Veteran Facing Forms Team

- Meeting 10/24
- Attendees: Amanda, Laura, Pat Siebenlist (PM), Rachel Shearer (UX Designer)
- Why Staging?
  - Looked into UXPin for a while - would work with screen readers, could import components. In the end this wasn't a great choice.
  - Previous team member did briefly look into Codepen (if we wanted Pat to look for these notes that might be a possibility)
  - Ultimately they went with staging
- Implementation of prototype
  - Testing new variants/extensions of existing components
  - Large lift, Would have been nice to get from blueprint to testing quicker
  - Went into testing with fully coded form not necessarily MVP
  - Not time wasted in their case but if there is a lot of experimental pieces that you are testing you might have to go backwards a bit
- Usability Testing:
  - Can report back on secondary findings as well since they used those (i.e. v3 components)
- Future thoughts
  - Case-by-case basis
  - Super open ended, meaning they aren’t limiting themselves to just a couple options
  - cost/benefit considerations

**Comment by laflannery** [comment]
### Check-In Experience Team

- Meeting 11/8
- Attendees: Amanda, Laura, Ben Brasso (UX Researcher), Christina Gednalske (UX Designer), John Woolschlager (FE Engineer)
- Why CodePen?
  - Christina's previous team had used CodePen so it was a tool were aware of
  - They were specifically looking to test accessibility
  - Didn’t explore staging because check-in can’t easily use staging
  - They were looking for a tool that would allow designers to be able to use it more easily and not require as much engineering resources/time
    - At one point, they looked at CodeSandbox but this would have made it a lot harder for designers to work independently and required more engineering assistance
- Implementation of prototype
  - Engineers built the initial set up but now designers do the implementation for each prototype - for the most part, sometimes very custom things need engineering support
  - All designers within the team have used it and they all don’t necessarily have HTML knowledge
  - Component code is able to be copied directly from Storybook and pasted into CodePen and it renders correctly (assuming the DS has been set up to work with it but we should be able to copy what Brian and John set up to do this as well)
- Usability Testing: Have done one round so far and it went well and did include participants that used screen readers
